include/struct_decls/struct_0203CDB0_decl.h

```diff
@@ -1,6 +1,6 @@
 #ifndef POKEPLATINUM_STRUCT_0203CDB0_DECL_H
 #define POKEPLATINUM_STRUCT_0203CDB0_DECL_H
 
-typedef struct UnkStruct_0203CDB0_t UnkStruct_0203CDB0;
+typedef struct UnkStruct_0203CDB0_t FieldSystem;
```

Comment:
Remove this file

Reply:
will double check in a second to make sure but if it's the same as hgss there are struct pointers in FieldSystem that also include pointers to FieldSystem, so this just defines FieldSystem to avoid the error caused by the mutual inclusion

Reply:
If it's needed then rename `UnkStruct_0203CDB0_t` and move the forward declaration to wherever the struct is defined and then remove this file

Reply:
marking as out of scope for this pr cause it wasn't meant to be field system focused, I just wanted to doc a file that used field system